framework/source/class/qx/event/handler/GestureCore.js

```diff
@@ -59,7 +59,31 @@ qx.Bootstrap.define("qx.event.handler.GestureCore", {
      * @type {Integer} Factor which is used for adapting the delta of the mouse wheel
      * event to the roll events,
      */
-    ROLL_FACTOR : 18
+    WHEEL_ROLL_FACTOR: 18,
```

Comment:
I'm not sure. This is just a comment so other reviewers could discuss this. To preserver compatibility it would be best to leave `qx.event.handler.GestureCore.ROLL_FACTOR` as is and use it and only add the new `qx.event.handler.GestureCore.TOUCHPAD_ROLL_FACTOR`.